lib/propertyTypes.js

```diff
@@ -2,12 +2,13 @@
  * @name propertyTypes
  * @constant
  * @memberof Helpers
- * @summary Checks matching via various operators
- * @returns {Boolean} the result of the check
+ * @summary Normalizes values out to various property types
```

Comment:
A more exact summary might be "Converts string values to other types"

Reply:
Updated